core/src/main/java/org/jdbi/v3/core/spi/package.html

```diff
@@ -0,0 +1,22 @@
+<!--
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<html><body>
+<p>
+The <code>spi</code> package includes entrypoints to extending <code>Jdbi</code>
```

Comment:
I'd rather change it now than regret it forever... but that's for a separate issue.